doc/jsk_pcl_ros/nodes/octomap_server_contact.md

```diff
@@ -16,6 +16,10 @@ You can pass the output of vision and contact sensors.
 
    input pointcloud.
 
+* `proximity_in` (`sensor_msgs/PointCloud2`)
```

Comment:
Is this PR waiting for https://github.com/jsk-ros-pkg/jsk_recognition/pull/2329, or independent from it?

Reply:
Independent from #2329